pull_request_template.md

```diff
@@ -0,0 +1,24 @@
+# # Description
+*Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.*
+
+**Fixes # (issue)**
+
+<your description here>
+
+## Type of change
+Please delete options that are not relevant.
+
+- Bug fix (non-breaking change which fixes an issue)
+- New feature (non-breaking change which adds functionality)
+- Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- This change requires a documentation update
+- UI Change (change that will effect the visual experience)
+
+# Checklist:
+- I have manually tested my changes on desktop and mobile
```

Comment:
should this portion be an actual checklist? so `- [ ] `

Reply:
Oh it totally should. It was rendering as a checkbox in my markdown editor for some reason. Thanks for seeing that!